hathor/p2p/states/ready.py

```diff
@@ -209,10 +208,11 @@ def handle_pong(self, payload: str) -> None:
         self.ping_start_time = None
         self.log.debug('rtt updated', rtt=self.ping_rtt, min_rtt=self.ping_min_rtt)
 
-    def send_get_best_blockchain(self, n_blocks: int = settings.DEFAULT_BEST_BLOCKCHAIN_BLOCKS) -> None:
+    def send_get_best_blockchain(self, n_blocks: Optional[int] = None) -> None:
         """ Send a GET-BEST-BLOCKCHAIN command, requesting a list of the latest
         N blocks from the best blockchain.
         """
+        n_blocks = n_blocks if n_blocks is not None else self._settings.DEFAULT_BEST_BLOCKCHAIN_BLOCKS
```

Comment:
Use explicit typing here: `n_blocks: int`.

Reply:
Changed in https://github.com/HathorNetwork/hathor-core/pull/770/commits/e2afcd970cff9597b7d5c5528b09e3a971870fc6. I had to change the variable name for the linter

Reply:
Thanks!